.github/workflows/springbok.yml

```diff
@@ -0,0 +1,18 @@
+name: Springbok README tests
+on:
+  push:
+  schedule:
+    - cron: '0 23 * * *' # run daily at 23:00 (UTC)
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
```

Comment:
Ubuntu Focal's CMake is on 3.16.3. It will fall on building IREE (CMake >= 3.17). Probably want to change the base to Ubuntu 22.04.

Reply:
virtual-environments side-loads latest `cmake` when building the ubuntu 20.04 image. https://github.com/actions/virtual-environments/blob/9f9633ce6f0385f75fd3a66db9a50c7902e4a686/images/linux/ubuntu2004.json#L205, so it runs on cmake 3.23 instead.